pkg/logging/logging.go

```diff
@@ -29,8 +29,8 @@ func InitLogger(logLevelString string, clusterID string) *zap.SugaredLogger {
 	config.EncoderConfig.TimeKey = "timestamp"
 	config.Level = logLevel
 	config.EncoderConfig.EncodeTime = zapcore.RFC3339TimeEncoder
-	config.EncoderConfig.StacktraceKey = "" // to hide stacktrace info
-	config.EncoderConfig.CallerKey = ""     // to hide caller info
+	config.EncoderConfig.StacktraceKey = ""   // to hide stacktrace info
+	config.EncoderConfig.CallerKey = "caller" // to hide caller info
```

Comment:
Question: Is the comment still correct with this change?

Reply:
Good catch, that's an artifact of the past :)